### PR TITLE
feature/30: Added Browser.write() function

### PR DESCRIPTION
**Pull request by EduAlberdi**
# PR Details
Added Browser.write() function in order to fake human writting by keyboard

## Related Issue

#30 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation 
- [ ] I have updated the documentation accordingly (README.md for each widget)
- [ ] I have added tests to cover my changes (at least 1 spec for each widget with the same coverage as the master branch)
- [x] All tests (new and existing) passed
- [ ] A new branch needs to be created from master to evolve previous versions
- [x] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [x] Add the issue into the right [project](https://github.com/systelab/systelab-components-wdio-test/projects) with the proper status (In progress)

### CODE REVIEW

File: src/wdio/browser.ts [hunk 1]
```diff
@@ -65,4 +65,9 @@ export class Browser {
     public static async waitUntil(condition: () => boolean | Promise<boolean>, timeout: number = DefaultTimeout.SLOW_WAIT): Promise<void> {
         await browser.waitUntil(condition, {timeout});
     }
+
+    //Keyboard functions
```
Comment (joaquimvila) [inline src/wdio/browser.ts:70]: Despite I am OK with the new method added, I would suggest the following to improve/polish the proposed implementation:
- Move this method together with the block of "keyboard control" functions (defined on the top of the class)
- Rename it to `writeText(text: string)`, as write may be too generic

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components-wdio-test",
-  "version": "1.5.0",
+  "version": "1.7.0",
```
Comment (joaquimvila) [inline package.json:3]: Do we really need to update twice the version number (considering PR #28)?
I would suggest to move again to version 1.6.0, so both changes are included on a single new NPM version

**Review by joaquimvila — APPROVED** [review]
Good job @EduAlberdi! Thanks for your contribution.

**Review by atzurk — APPROVED** [review]
Thanks Edu! This will help us to deal with the text input and focus management for those text fields embedded in grids.

**Pull request merged**